[docs] Document the workaround for crashing a translated page (#35720)
mui/material-ui: docs/data/material/components/buttons/buttons.md

{{"demo": "LoadingButtonsTransition.js"}}

:::warning

> There is a [known issue](https://github.com/mui/material-ui/issues/27853) with translating the page with Chrome tools when Loading Buttons are present.

Reviewer: ```suggestion
There is a [known issue](https://github.com/mui/material-ui/issues/27853) with translating the page using Chrome tools when Loading Buttons are present.
```

After the page is translated, the application crashes when the loading state of a Button changes.

> To prevent it, ensure that the contents of the Loading Button is inside any HTML element (a `span` will suffice):

Reviewer: ```suggestion
To prevent this, ensure that the contents of the Loading Button are nested inside any HTML element, such as a `<span>`:
```
